src/akkd/Accordion/AccordionTitle.tsx

```diff
@@ -0,0 +1,26 @@
+import classNames from 'classnames'
+import React from 'react'
+import styles from './Accordion.module.css'
+
+interface Props {
+	index: any,

```

Comment:
Given index is a straight pass-through, never used, this could naively be types as `unknown`.

To go even more accurate, though, you can genericise the interface with `index: T`, and `onClick?: (event: React.MouseEvent, props: {index: T}) => void` - that way you get a tight typing, but the consumer can use whatever it wants as an index